project/src/main/webapp/shared.js

```diff
@@ -269,13 +282,9 @@ function createRandomColorBlock(elementId) {
 
 /** Pick random color from Bootstrap defaults */
 function pickRandomColorClass() {
-  const colorClasses =
-      ['bg-primary',
-       'bg-success',
-       'bg-info',
-       'bg-warning',
-       'bg-danger']
-  return colorClasses[Math.floor(Math.random() * colorClasses.length)]
+  const colorClasses = [
+    'bg-primary', 'bg-success', 'bg-info', 'bg-warning', 'bg-danger'
+  ] return colorClasses[Math.floor(Math.random() * colorClasses.length)]
```

Comment:
add semicolon (my bad, didn't have one to start with) and new line the return statement

Reply:
done.